src/pages/LandingPages/MGCovid19/MGCovidExplained.vue

```diff
@@ -45,16 +45,16 @@
 							loading="lazy"
 						>
 					</div>
-					<h3 class="three-steps__title">
+					<h3 class="tw-mb-1">
 						Check
 					</h3>
-					<div class="three-steps__description">
+					<div class="text-center">
```

Comment:
Should this be tw-text-center as well?

Reply:
Yes, I should do a find-replace on these.

Reply:
It works because of a foundation util class right now I think.

Reply:
👍